packages/stores/src/account/base.ts

```diff
@@ -733,6 +733,12 @@ export class AccountStore<Injects extends Record<string, any>[] = []> {
       throw new Error("Failed to retrieve account from signer");
     }
 
+    // If the memo is empty, set it to "FE" so we know it originated from the frontend for
+    // QA purposes.
+    if (memo === "") {
+      memo = "FE";
+    }
```

Comment:
Could we append it to the bottom of the memo if the memo is present? 

At the same time, it might make sense to make it clear to users that this metadata instrumentation is done by our app to avoid confusion. E.g. append the following:
"The flag below is appended by the Osmosis frontend\nFE"

Reply:
Agreed offline we want the memo field to be super lean to not increase any on chain storage.

Reply:
Added memo append logic here https://github.com/osmosis-labs/osmosis-frontend/pull/2757/commits/3e191db4f56d111b074569c36e3961965a6345a2